components/elm/src/main/elm_instance.F90

```diff
@@ -0,0 +1,31 @@
+module elm_instance
+
+use seq_comm_mct, only: seq_comm_suffix, seq_comm_inst, seq_comm_name
+
+implicit none
+private
+save
+
+public :: elm_instance_init
+
+integer,           public :: lnd_id
+integer,           public :: inst_index
+character(len=16), public :: inst_name
+character(len=16), public :: inst_suffix
```

Comment:
It would be useful to added comment to describe what these variables. e.g. `lnd_id` based on https://github.com/E3SM-Project/scream/blob/master/driver-mct/shr/seq_cdata_mod.F90#L49 is `component ID (in the coupler?)`

Reply:
Comment added for lnd_id in the new commit. Thanks.